system/security/cryptohelper/digisign.cpp

```diff
@@ -136,6 +138,22 @@ class CDigitalSignatureManager : public CSimpleInterfaceOf<IDigitalSignatureMana
     {
         return digiVerify(b64Signature, strlen(text), text);
     }
+
+    void setKeyFileNames(const char * pub, const char * priv)
```

Comment:
With Jake's refactoring this should be obtainable from the pubKey/privKey class, so no need to set up like this.

Reply:
I think his changes still mean you can remove this call.  E.g. You can implement the function as:

    virtual const char * queryPublicKeyFile() const override
    {
        return pubKey->queryKeyName();
    }